src/modules/comment-pop-up-structure.js

```diff
@@ -86,7 +86,7 @@ const commentPopUp = (obj, id, getCommentCallback, setCommentCallback) => {
     (data) => data,
     (error) => {
       throw new Error(error);
-    }
+    },
   );
 
```

Comment:
- try to modularize your codes for better code maintanablity.

Reply:
thanks